.github/workflows/test.yml

```diff
@@ -10,7 +10,20 @@ concurrency:
 
 jobs:
   tests:
-    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        target:
+          - id: "linux-aarch64"
+            os: "ubuntu-latest"
```

Comment:
I don't think changing Ubuntu version to latest make the workflow run on an aarch64 machine?